transactions/stx.ts

```diff
@@ -151,8 +150,6 @@ export async function generateUnsignedSTXTokenTransfer(
     fee: 0,
     sponsored,
     anchorMode: anchorMode ? anchorMode : AnchorMode.Any,
-    postConditionMode,
-    postConditions,
   };
```

Comment:
it doesn't look like this function is used in either xverse-web-extension or xverse. can we mark it with @deprecated?